core/create_config.py

```diff
@@ -98,28 +88,26 @@ def generate_validator_config(dev: bool = False) -> Dict[str, Any]:
         or None
     )
 
-    wallet_name = input(
-        "💼 Enter wallet name (default: default): ") or "default"
-    hotkey_name = input(
-        "🔑 Enter hotkey name (default: default): ") or "default"
+    wallet_name = input("💼 Enter wallet name (default: default): ") or "default"
+    hotkey_name = input("🔑 Enter hotkey name (default: default): ") or "default"
     netuid = 241 if subtensor_network.strip() == "test" else 56
     postgres_user = "user"
-    postgres_password = generate_secure_password(
-    ) if not postgres_password else postgres_password
+    postgres_password = generate_secure_password() if not postgres_password else postgres_password
     postgres_db = "god-db"
     postgres_host = "localhost"
     postgres_port = "5432"
 
-    validator_port = input(
-        "👀 Enter an exposed port to run the validator on (default: 9001): ") or "9001"
+    validator_port = input("👀 Enter an exposed port to run the validator on (default: 9001): ") or "9001"
+
+    gpu_ids = input("🎮 Enter comma-separated GPU IDs to use (e.g., 0,1,2, default = 0): ").strip() or "0"
 
     s3_compatible_endpoint = input("🎯 Enter s3 compatible endpoint: ")
     s3_compatible_access_key = input("🎯 Enter s3 compatible access key: ")
     s3_compatible_secret_key = input("🎯 Enter s3 compatible secret key: ")
     s3_bucket_name = input("🎯 Enter your s3 bucket name: ")
+    s3_region = input("🎯 Enter s3 region (default: us-east-1): ") or "us-east-1"
```

Comment:
What happens if people use backblaze or non aws providers? Should they put us-east-1? is it clear from the docs?